CompetitionMARS/Tests/CertificationTests.cs

```diff
@@ -0,0 +1,364 @@
+﻿
+using AventStack.ExtentReports.Reporter;
+using AventStack.ExtentReports;
+using CompetitionMARS.Models;
+using CompetitionMARS.Pages;
+using CompetitionMARS.Utilities;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using OpenQA.Selenium.Support.UI;
+using OpenQA.Selenium;
+using SeleniumExtras.WaitHelpers;
+using NUnit.Framework;
+using Microsoft.VisualStudio.TestPlatform.ObjectModel;
+
+namespace CompetitionMARS.Tests
+{
+    public class CertificationTests : CommonDriver
+    {
+        private Certification certPage;
+        public static ExtentReports extent;
+        public List<string> addedCertificationsEntries = new List<string>();
+
+
+        [OneTimeSetUp]
+        public void SetUp()
+        {
+            // Start the report
+            extent = ExtentManager.StartReport();
```

Comment:
also when you move this to common place you dont have to keep doing the same code again! ideally extent reporter is initialised once, hence why its a onetimesetup hook. right now its pointless as you have it in all your classes